packages/earn-controller/src/EarnController.ts

```diff
@@ -436,54 +576,80 @@ export class EarnController extends BaseController<
    * Updates the vault metadata in the controller state including APY, capacity,
    * fee percentage, total assets, and vault address.
    *
+   * @param chainId - The chain id to refresh pooled staking vault metadata for (optional).
    * @returns A promise that resolves when the vault metadata has been updated
    */
-  async refreshPooledStakingVaultMetadata(): Promise<void> {
-    const chainId = this.#getCurrentChainId();
-    const vaultMetadata = await this.#stakingApiService.getVaultData(chainId);
+  async refreshPooledStakingVaultMetadata(chainId?: number): Promise<void> {
```

Comment:
Since this is a public method perhaps we ought to document that this now takes an optional `chainId` method?